### PR TITLE
OCPBUGS#12151: oc get packagemanifests procedure step

### PR DESCRIPTION
**Pull request by michaelryanpeter**
Version(s): ~4.10+~ 4.13+

Issue: [OCPBUGS-12151](https://issues.redhat.com/browse/OCPBUGS-12151)

Link to docs preview: 
* Admin tasks > [Installing a specific Operator](https://59876--docspreview.netlify.app/openshift-enterprise/latest/operators/admin/olm-adding-operators-to-cluster.html#olm-installing-specific-version-cli_olm-adding-operators-to-a-cluster)
* User tasks > [Installing Operators in your namespace ](https://59876--docspreview.netlify.app/openshift-enterprise/latest/operators/user/olm-installing-operators-in-namespace.html#olm-installing-specific-version-cli_olm-installing-operators-in-namespace)

QE review:
- [ ] QE has approved this change.

Additional information:

Originally, this was scoped for 4.10+, but the information architecture in 4.12 and below does not support this update in the procedure. I will create a new ticket to track this update in 4.10-4.12

### CODE REVIEW

File: modules/olm-installing-specific-version-cli.adoc [hunk 1]
```diff
@@ -123,6 +123,21 @@ You can format an Operator's version and channel information in the YAML format
 $ oc get packagemanifests <operator_name> -n <catalog_namespace> -o yaml
 ----
 ====
++
+Running the `oc get packagemanifest` or `oc describe packagemanifest` commands can return a random package manifest if the following conditions are met:
```
Comment (kevinrizza) [inline modules/olm-installing-specific-version-cli.adoc:127]: I would maybe rephrase

`return a random package manifest`

to

`return a package from a nondeterministic catalog source` ?

It's not that any random package will show up, it's just that the package may be from a source you may not expect if there's more than one package with the same name. Generally this means that you're going to sometimes get the community version of an operator instead of the certified one, or vice versa.

**Comment by kalexand-rh** [comment]
The `branch/enterprise-4.14` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.13. And any PR going into `main` must also target the latest version branch (enterprise-4.14).

If the update in your PR does NOT apply to version 4.14 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

**Review by anik120 — COMMENTED** [review]
/lgtm

**Comment by kcarmichael08** [comment]
Nice! LGTM!

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done
